pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -88,7 +90,20 @@ func (v *Venafi) Sign(ctx context.Context, cr *cmapi.CertificateRequest, issuerO
 
 	duration := apiutil.DefaultCertDuration(cr.Spec.Duration)
 
-	certPem, err := client.Sign(cr.Spec.CSRPEM, duration)
+	customFields := []certificate.CustomField{}
+	if annotation, exists := cr.GetAnnotations()["venafi.cert-manager.io/custom-fields"]; exists && annotation != "" {
+		err := json.Unmarshal([]byte(annotation), &customFields)
+		if err != nil {
+			message := "Failed to parse venafi.cert-manager.io/custom-fields anotation"
+
+			v.reporter.Pending(cr, err, "VenafiCustomFieldsParseError", message)
```

Comment:
`CustomFieldsError` I think is sufficient

Reply:
This should also be `Failed` not `Pending` - we don't support these fields changing after creation 😄